workloads/virtual-machines/expose-service.md

```diff
@@ -1,9 +1,94 @@
 # Expose Service
```

Comment:
should we make networking section where DNS and Expose Service are separate sections under Networking?  There are other aspects of networking that need to be documented as well that probably belong in a top level networking section.

Reply:
Yes, makes sense.

Reply:
Done